stylesheet.css

```diff
@@ -1,11 +1,75 @@
 html {
     box-sizing: border-box;
+    font-family: 'Source Sans Pro';
```

Comment:
I would always add a `sans-serif` as backup if the font-family does not load. You can read more about it on MDN:
https://developer.mozilla.org/en-US/docs/Web/CSS/font-family